addon/components/ember-selectize.js

```diff
@@ -107,6 +107,11 @@ export default Ember.Component.extend({
   selectizeOptions: Ember.computed(function() {
     var allowCreate = get(this, 'create');
 
+    //Split the passed in plugin config into an array.
+    if (typeof this.plugins === 'string') {
+      this.plugins = this.plugins === "" ? [] : this.plugins.split(',');
```

Comment:
What if me `map` this with [trim](http://www.w3schools.com/jsref/jsref_trim_string.asp)?